draft-ietf-rats-corim.md

```diff
@@ -387,8 +387,22 @@ The following describes each child element of this type.
 
 ### Profile Types {#sec-corim-profile-types}
 
-A profile specifies which of the optional parts of a CoRIM are required, which
-are prohibited and which extension points are exercised and how.
+Profiling is the mechanism that allows the base CoRIM schema to be customised to fit a specific Attester.
+
+A profile defines which of the optional parts of a CoRIM are required,
+which are prohibited and which extension points are exercised and how.
+A profile MUST NOT alter the syntax or semantics of an already defined CoRIM type.
+A profile MAY constrain the values of a given CoRIM type to a subset of the values.
+A profile MAY extend the set of a given CoRIM type using the defined extension points (see {{sec-extensibility}}).
+Exercised extension points should preserve the intent of the original semantics.
+
+CoRIM profiles SHOULD be specified in a publicly available document.
+
+A CoRIM profile can use one of the base CoRIM media types defined in {{sec-mt-corim-signed}} and
+{{sec-mt-corim-unsigned}} with the `profile` parameter set to the appropriate value.
+Alternatively, it MAY define and register its own media type.
+
+A profile identifier is either an OID {{-cbor-oids}} or a URL {{-uri}}.
```

Comment:
A profile is fixed once it is released, any subsequent change will lead to a new profile version.

Reply:
done in [48d4739](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/76/commits/48d47398efef6db443417ce410811ef6bae23d9b)